libvast/include/vast/synopsis.hpp

```diff
@@ -73,11 +78,7 @@ class synopsis {
 
   // -- serialization ----------------------------------------------------------
 
-  /// Saves the contents (excluding the layout!) of this slice to `sink`.
-  virtual caf::error serialize(caf::serializer& sink) const = 0;
-
-  /// Loads the contents for this slice from `source`.
-  virtual caf::error deserialize(caf::deserializer& source) = 0;
+  virtual caf::error inspect(supported_inspectors& inspector) = 0;
```

Comment:
I think it's rather dangerous to name this function `inspect`, as that usually means it's being called directly by VAST.

Reply:
Maybe something like `inspect_impl` or so is better here, just to make it obvious that it is being called from the actual `inspect` overload.

Reply:
I will rename it in my next PR